nautobot/dcim/migrations/0060_module_models.py

```diff
@@ -455,6 +456,10 @@ class Migration(migrations.Migration):
             name="rearport",
             options={"ordering": ("device", "module", "_name")},
         ),
+        migrations.AlterModelOptions(
+            name="interface",
+            options={"ordering": ("device", "module", nautobot.core.models.query_functions.CollateAsChar("_name"))},
+        ),
```

Comment:
Out of curiosity, do you know/understand why is `interface` the only model using `nautobot.core.models.query_functions.CollateAsChar()` compared to all of the other components?

Reply:
No idea why but it would be nice to have natural sorting on all component models to avoid this:

![image](https://github.com/nautobot/nautobot/assets/75227981/ac650d59-6a58-4bd8-bfa0-ee02347170b0)

Reply:
The above screenshot was resolved when I fixed the missing `NaturalOrderingField` on `ModuleBay`. I'm not sure what the `CollateAsChar` is doing